package.json

```diff
@@ -52,10 +60,10 @@
     "typescript": "^3.9.3"
   },
   "dependencies": {
-    "@grpc/proto-loader": "^0.5.4",
     "@types/google-protobuf": "^3.7.2",
+    "@types/wtfnode": "^0.7.0",
     "google-protobuf": "^3.12.2",
     "grpc": "^1.24.2",
-    "grpc-tools": "^1.9.0"
+    "wtfnode": "^0.8.1"
```

Comment:
Is this actually used? I couldn't spot it

Reply:
Yep was using it for debugging, thanks removed it.